src/python/pants/backend/jvm/subsystems/scoverage_platform.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+import logging
+import os
+from pants.build_graph.injectables_mixin import InjectablesMixin
+from pants.subsystem.subsystem import Subsystem
+from pants.java.jar.jar_dependency import JarDependency
+from pants.backend.jvm.targets.jar_library import JarLibrary
+from pants.build_graph.address import Address
+
+logger = logging.getLogger(__name__)
+
+SCOVERAGE = "scoverage"
+
+class ScoveragePlatform(InjectablesMixin, Subsystem):
+  """The scoverage platform."""
+
+  options_scope = 'scoverage'
+
+  @classmethod
+  def register_options(cls, register):
+    super().register_options(register)
+    register('--enable-scoverage',
+      default=False,
+      type=bool,
+      help='Specifies whether to generate scoverage reports for scala test targets.'
+           'Default value is False. If True,'
+           'implies --test-junit-coverage-processor=scoverage.')
+
+    register('--blacklist-file',
```

Comment:
Pants supports `@file` options natively, so you should make this option a list option:
```
register('--blacklist-targets',
  type=list,
  member_type=target_option,
  ..
)
```

And then someone who wants to put the list of targets in a file (in python list syntax) can pass `--blacklist-targets=@some-blacklist-file`.

You could also pass in a (string) list of regexes to match against target names.